index.rst

```diff
@@ -128,7 +129,7 @@ The CSC is allowed to publish as many events as necessary to convey the informat
 
 .. _section-other-settings-applied:
 
-Other Settings Applied events
+Other settingsApplied Events
```

Comment:
Same as before, why use this format? Note that we are not referring to "the" `settingsApplied` event here...

Reply:
I made it consistent with the settingsVersion heading (which i think is line ~66). 
I switched them all back to non-camel case for the upcoming revamp... I don't think it really matters much so long as it's all consistent.